jmclient/jmclient/configure.py

```diff
@@ -229,8 +229,9 @@ def jm_single():
 # N=1, so we choose 3 for a more reasonable figure, as our default.
 # You can also set your own fee/kb: any number higher than 1000 will
 # be interpreted as the fee in satoshi per kB that you wish to use
-# example: N=30000 will use 30000 sat/kB as a fee, while N=5
-# will use the estimate from your selected blockchain source
+#
+# Example: N=30000 will use 30,000 sat/kB (30 sat/B) as a fee, while
```

Comment:
Not sure comma should be used as thousands separator, in most of the countries of the world it's decimal separator, so might be confusing for some people.

Also, since segwit it's tehnically sat/kvB (we use sat/vkB in JM code somewhere, which I think is not correct, but that's my fault historically) and sav/vB.

Reply:
There seems to be three viable solutions for a thousands separator.

> Space, the internationally recommended thousands separator.
https://en.wikipedia.org/wiki/Decimal_separator

> 10_000_000.0
https://www.python.org/dev/peps/pep-0515/

And apostrophes the way C++ does it
https://en.cppreference.com/w/cpp/language/integer_literal

Using underscores seems like the worst of the picks.  But it is standard Python so it could be useful.  Currently it does not work in the configuration.

 Using spaces works fine in this scenarios: `30 000`, but I think that the C++ way is less shocking to the most amount of people: `30'000`.  It has the added benefit of no causing confusion for smaller numbers.  Consider the following:
1. It might use any value between 8'000 and 12`000 for your transactions.
2. It might use any value between 8 000 and 12 000 for your transactions.
3. It might use any value between 8000 and 12000 for your transactions.

Both `8000` and `8 000` require more focus to understand than `8'000` in my opinion.  Any thoughts?

Reply:
I don't think almost anyone is familiar with apostrophe as a thousands separator; according to [Wikipedia](https://en.wikipedia.org/wiki/Decimal_separator#Examples_of_use) this is only used in Switzerland and sometimes Liechtenstein and Italy, and even there only in certain cases(?), so I would not suggest doing this, and would suggest having no separator, to avoid the localization issue. Yes it is not *very* easy to read but it's better than using a notation people don't understand, which will really confuse them.